storage-api/pom.xml

```diff
@@ -177,6 +177,7 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
+        <version>3.0.0-M4</version>
```

Comment:
Should we add this version in the properties block too?

Reply:
We could, but it is only used in the single place.

Reply:
I mostly use properties for versions that get referenced multiple times, but if you want me to change it I will.

Reply:
I don't think it's a major issue, I just find it easier to have all versions together in the properties block and I also thought this was the approach taken by Hive in general (I do not think this is formalized anywhere, just based on previous changes made to the top pom.xml file). I'll leave it up to you.